src/controller/formController.js

```diff
@@ -0,0 +1,33 @@
+  app1.controller("formController",function($scope,createFactory){
+   
+    $scope.event_type="art";
+     $scope.myFunc=function(){
+      $scope.data={
+      'event_title':  $scope.event_title,
+      'event_date':  $scope.event_date.toISOString().slice(0, 10)+" "+ $scope.event_time.toISOString().slice(12, 19),
+
+      'event_venue':  $scope.event_venue,
+      'event_type':  $scope.event_type,
+      'event_author': "sdfjhsjf",
+      'event_price':  $scope.event_price,
+      'event_description':  $scope.event_description
+
+    };
+    $scope.getid;
+
+     console.log( $scope.data);
+     console.log($scope.event_time.toISOString().slice(12, 19));
+
+     createFactory.all( $scope.data).then(
```

Comment:
please change the function names, follow nouns for variables and verbs for function names
https://github.com/practo/javascript/blob/master/guides/coding-standards/coding-standards.md